changelogs/changelog.yaml

```diff
@@ -1971,3 +1971,59 @@ releases:
           created using the same name as an existing uplink, then the existing
           uplink is modified.
     release_date: '2024-07-31'
+
+  9.6.0:
+    changes:
+      release_summary: '- The ``ome_application_certificate`` module is
+          enhanced to support the upload of certificate chain.
+
+        - The ``idrac_support_assist`` module is added to add support
+          to run and export SupportAssist collection logs on iDRAC.
+
+        - The ``idrac_secure_boot`` module is added to import the secure
+          boot certificate.'
+
+      major_changes:
+        - idrac_support_assist - This module allows to run and export
+          SupportAssist collection logs on iDRAC.
+        - idrac_secure_boot - This module allows to import the secure boot
+          certificate.
+
+
+      minor_changes:
+        - ome_application_certificate - This module is enhanced to support
```

Comment:
This module is enhanced to support the upload operation of one or more certificates on OpenManage Enterprise.

Reply:
Existing is fine.